ChangeLog

```diff
@@ -49,6 +49,7 @@
 	* resume data no longer has timestamps of files
 	* require C++11 to build libtorrent
 
+	* fix crash caused by empty bitfield 
```

Comment:
I didn't notice this until just now. this is in the wrong place. you're in master, this should be at the top of the fil